src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  width: 166px;
+  height: 358px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  color: $main-text-color;
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+
+  cursor: pointer;
+  transition: transform 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+    color: #34568b;
+  }
+
+  &__photo {
+    background-image: url('../images/imac.jpeg');
+    width: 160px;
+    height: 134px;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-style: normal;
+    font-size: 12px;
+    line-height: 18px;
+    text-decoration: none;
+    color: $main-text-color;
+    margin-top: 40px;
+  }
+
+  &__name:hover {
+    color: #34568b;
+  }
+
+  &__product {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $second-text-color;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 16px;
+    margin-top: 16px;
+  }
+
+  &__button {
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    background-color: #00acdc;
+    border: 1px solid #00acdc;
+    margin-top: 16px;
+
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    text-transform: uppercase;
+    text-decoration: none;
+    text-align: center;
+    color: #fff;
+  }
+
+  &__button:hover {
+    color: #00acdc;
+    background-color: #fff;
+    cursor: pointer;
+  }
+}
+
+.reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-decoration: none;
+  color: $main-text-color;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+}
+
+.price__name {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: $second-text-color;
+}
+
+.price__number {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
```

Comment:
The line-height of the price number is the same as its font-size. This may lead to the text not being vertically centered in its container. Please, check this.